src/jest.mli

```diff
@@ -31,15 +31,15 @@ module Runner (A : Asserter) : sig
     val testAsync : string -> ?timeout:int -> ((_ A.t -> unit) -> unit) -> unit
     val testPromise : string -> ?timeout:int -> (unit -> _ A.t Js.Promise.t) -> unit
     val testAll : string -> 'a list -> ('a -> _ A.t) -> unit
-    external describe : string -> (unit -> unit) -> unit = "describe.only" [@@bs.val]
+    external describe : string -> (unit -> unit [@bs.uncurry]) -> unit = "describe.only" [@@bs.val]
   end
 
   module Skip : sig
     val test : string -> (unit -> _ A.t) -> unit
     val testAsync : string -> ?timeout:int -> ((_ A.t -> unit) -> unit) -> unit
     val testPromise : string -> ?timeout:int -> (unit -> _ A.t Js.Promise.t) -> unit
     val testAll : string -> 'a list -> ('a -> _ A.t) -> unit
-    val describe : string -> (unit -> unit) -> unit
+    external describe : string -> (unit -> unit [@bs.uncurry]) -> unit = "describe.skip" [@@bs.val]
```

Comment:
Could you explain this change? It was changed not long ago in https://github.com/glennsl/bs-jest/commit/339cac05701947efa3506ae456c445b55f3a8e97#diff-ea2f2fc3c806d2446d8c40ec58d9bd25R42 to fix a bug with Jest 23

Reply:
I think this one could be reverted. I think I was just trying to make it more consistent with the others.

Reply:
I see why you did this now. `@bs.uncurry` is only valid for `external`s. It shouldn't matter here though, since the test is skipped and the function shouldn't be executed anyway.

Reply:
Unfortunately it didn't seem to matter a great deal for the coverage either.